content/en/docs/reference/command-line-tools-reference/feature-gates/image-maximum-gc-age.md

```diff
@@ -6,8 +6,8 @@ _build:
   render: false
 
 stages:
-  - stage: alpha 
-    defaultValue: false
-    fromVersion: "1.29"   
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
```

Comment:
The changes made to the "stages" block appear incorrect. Please refer to the documentation [here](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates) for further information. Kindly consider the suggested correction provided below.

```suggestion
  - stage: alpha
    defaultValue: false
    fromVersion: "1.29"
    toVersion: "1.29"
  - stage: beta
    defaultValue: true
    fromVersion: "1.30"
```